pkg/tracer/event_common.go

```diff
@@ -27,28 +27,30 @@ func (e EventType) String() string {
 	}
 }
 
+// TcpV4 represents a TCP event (connect, accept or close) on IPv4
 type TcpV4 struct {
-	Timestamp uint64
-	CPU       uint64
-	Type      EventType
-	Pid       uint32
-	Comm      string
-	SAddr     net.IP
-	DAddr     net.IP
-	SPort     uint16
-	DPort     uint16
-	NetNS     uint32
+	Timestamp uint64    // Monotonic timestamp
```

Comment:
Is `godoc` able to handle such structuring of comments?

AFAIK, this is a common way to document fields in Go:

```
// CPU index
CPU uint64
```